### PR TITLE
Adjusting title to include both items

### PR DESCRIPTION
**Pull request by bergerhoffer**
Version(s):
4.12

Issue:
https://issues.redhat.com/browse/OSDOCS-4418

Link to docs preview:
https://54717--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-12-release-notes.html#ocp-4-12-empty-file-support-removed

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-12-release-notes.adoc [hunk 1]
```diff
@@ -1740,8 +1740,9 @@ Kubernetes 1.25 removed the following deprecated APIs, so you must migrate manif
 1. For more information about pod security admission in {product-title}, see xref:../authentication/understanding-and-managing-pod-security-admission.adoc#understanding-and-managing-pod-security-admission[Understanding and managing pod security admission].
 
 [id="ocp-4-12-empty-file-support-removed"]
-==== oc registry command for empty file support
-The `--registry-config` and `--to option` options in the `oc registry login` command will stop accepting empty files. These options will continue to work with files that do not exist. The ability to write output to `-`(stdout) is also removed.
+==== Empty file and stdout support for the oc registry login command
```
Comment (JoeAldinger) [inline release_notes/ocp-4-12-release-notes.adoc:1743]: /lgtm

**Comment by bergerhoffer** [comment]
@kalexand-rh This is a tweak based on SME feedback of one of the RN tracker items. Can you please approve this as an exception for merging today?

**Review by kalexand-rh — APPROVED** [review]
Merge exception granted.

**Comment by bergerhoffer** [comment]
Merging!

**Pull request merged**